src/backend/crew_ai/crew.py

```
@@ -107,7 +107,7 @@ def extract_url_from_query(query: str) -> str:
     return "website mentioned in query"
 
 
-def run_crew(query: str, model_provider: str, model_name: str, library_type: str = None, workflow_id: str = "", progress_queue=None):
+def run_crew(query: str, model_provider: str, model_name: str, library_type: str = None, workflow_id: str = "", progress_queue=None, org_id: str | None = None):
```

Ruff (0.15.18)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)